### ISSUE TITLE
Support Palette Mode

### ISSUE THREAD

**Issue by ramack** [Pes8/android-material-color-picker-dialog#24]
I suggest to add a feature to show a predefined list of colors, as in many cases the users do not need one very specific color, but selecting from some is often sufficient. Only in some cases it may be needed to adjust the color exactly.

**Comment by Pes8** [comment]
Yeas, it's a good idea but basically this color picker is born to let user choose a specific color. (It was my case :) ).

Right now I haven't time to work on this. If someone wants to collaborate and do it I'll merge it of course!

I'm sorry but I'm very busy in this period, maybe in 6 months I'll have some free time.

**Comment by ramack** [comment]
Fair enough. It was still the reason why I chose your picker, as it is simple, not obviously unmaintained and it let's the user select the color individually.

If time permits and and the palette comes to a priority, I will try to look into it and see whether I can submit a PR.

Thanks for all your efforts!
